config.go

```diff
@@ -93,6 +94,7 @@ type config struct {
 	PoolFees            float64 `long:"poolfees" description:"The per-ticket fee mandated by the ticket pool as a percent (e.g. 1.00 for 1.00% fee)"`
 	AddrIdxScanLen      int     `long:"addridxscanlen" description:"The width of the scan for last used addresses on wallet restore and start up (default: 750)"`
 	StakePoolColdExtKey string  `long:"stakepoolcoldextkey" description:"Enables the wallet as a stake pool with an extended key in the format of \"xpub...:index\" to derive cold wallet addresses to send fees to"`
+	AllowHighFees       bool    `long:"allowhighfees" description:"Force dcrd to allow highfees from all sendrawtransaction"`
```

Comment:
This is actually the RPC client doing this. Maybe "Force the RPC client to use the 'allowHighFees' flag when sending transactions'